alws/schemas/build_schema.py

```diff
@@ -338,12 +338,12 @@ def compare_module_data(
 async def _get_module_ref(
     component_name: str,
     modified_list: list,
-    platform_prefix_list: list,
+    platform_prefix_list: dict,
     module: ModuleWrapper,
     gitea_client: GiteaClient,
     devel_module: typing.Optional[ModuleWrapper],
     platform_packages_git: str,
-    beholder_data: typing.List[dict],
+    beholder_data: tuple[typing.Any],
```

Comment:
I'm not sure, but I think that we might face an error on the pulp host with `tuple` annotation

Reply:
`asyncio.gather` returns `tuple[Any]`
![image](https://github.com/AlmaLinux/albs-web-server/assets/50624082/599a3cf8-63a2-4d42-8dd5-5d8f3b1687a4)

Reply:
That's not what I meant, I meant that the pulp host has a lower python version than the web server and we might face an error with builtin type annotation, but that's just a guess

Reply:
Doesn't matter pulp host's interpreter of python, because that code is executed under py3.9 on a host of web server